cmd/thanos/rule.go

```diff
@@ -758,3 +738,47 @@ func addDiscoveryGroups(g *run.Group, c *http_util.Client, interval time.Duratio
 		cancel()
 	})
 }
+
+type errInvalidPattern struct {
+	err error
+	pat string
+}
+
+func (e *errInvalidPattern) Error() string {
+	return errors.Wrapf(e.err, "retrieving rule files failed. Ignoring file. pattern %s", e.pat).Error()
+}
```

Comment:
Missing white space after function body.

Reply:
@kakkoyun  Added a newline. If you think OK, please help to resovle this conversation.

Reply:
All good, but I also don't see a button to resolve conversation.

Reply:
@kakkoyun Under the convsersation, there's a button. Maybe only the submitter can see it. Will click the button.